requirements.txt

```diff
@@ -27,12 +27,12 @@ git+https://git@github.com/uc-cdis/cdis_oauth2client.git@chore/py3#egg=cdis_oaut
 cdispyutils==0.2.13
 git+https://git@github.com/uc-cdis/datamodelutils.git@chore/py3#egg=datamodelutils
 # TODO:update to pypi version once released
-git+https://git@github.com/uc-cdis/psqlgraph.git@chore/py3#egg=psqlgraph
+git+https://git@github.com/uc-cdis/psqlgraph.git@develop#egg=psqlgraph
```

Comment:
I'm actually surprised this works given sheepdog on `master` is not python3 yet but if the tests pass...

If you can pin these to the versions from pypi instead of using the vcs branches, it'd make it a lot easier for me to resolve conflicts when I merge the python3 branch in.

psqlgraph==3.0.0 https://github.com/uc-cdis/psqlgraph/releases/tag/3.0.0
gen3datamodel=3.0.0 https://github.com/uc-cdis/gdcdatamodel/releases/tag/3.0.0

Reply:
Awesome, thank you for pointing out the pypi packages! I'll pin those. I'm merging onto `chore/python3` so I think that's why it's working with the develop branches here (?)

Reply:
That makes sense, sounds great!